less/index.less

```diff
@@ -53,7 +53,145 @@ table {
     box-sizing: border-box;
 }
 
-html, body {
-    height: 100%;
-}
+	html, body {
+		height: 100%;
+		font-size: 62.5%;
+	
+	}
+
+
+	
+
+	//Variable
+	@main-background:url(../steel.jpg);
+	@text-color:black(20, 19, 19, 0.945);
+	@font:'Audiowide', cursive;
+	//Mixins
+	.round-borders{
+		width:7rem;
+		border-radius: 100%;
+		border:1px solid @text-color;
+		margin-right: 1rem;
+	}
+	.border{
+		border:2px solid black;
+	}
+
+	//start of my css
+
+	.container{
+		background-image: @main-background;
+		background-repeat: no-repeat;
+		background-size: cover;
+		margin:auto;
+		max-width: 880px;
+		border: 2px rgba(0, 0, 0, 0.24) solid;
+		font-size: 2rem;
+		font-family: @font;
+		margin: 10px;
```

Comment:
Be careful - you've set margin twice here for the `.container` selector:
```
margin: auto;
margin: 10px;
```

The second setting for margin will overwrite `margin:auto` and add a 10px margin for all sides of your container. I would suggest setting the `margin` property only once with two values, like so:

```
margin: 10px auto; // 10px for top & bottom; auto (centered) for left & right
```